pkg/operator/api.go

```diff
@@ -30,6 +30,18 @@ const (
 	// used to determine which CDI instance is "active"
 	// and maybe other stuff some day in the future
 	ConfigMapName = "cdi-config"
+	//ManagedByLabel This label will be set on all resources created by the operator
+	ManagedByLabel = "app.kubernetes.io/managed-by"
+	//ManagedByLabelOperatorValue This label will be set on all resources created by the operator
+	ManagedByLabelOperatorValue = "cdi-operator"
+	//InstallStrategyVersionAnnotation This annotation represents the kubevirt version for an install strategy configmap.
+	InstallStrategyVersionAnnotation = "cdi.kubevirt.io/install-strategy-version"
+	//InstallStrategyRegistryAnnotation This annotation represents the kubevirt registry used for an install strategy configmap.
+	InstallStrategyRegistryAnnotation = "cdi.kubevirt.io/install-strategy-registry"
+	//EphemeralBackupObject This annotation represents that this object is for temporary use during updates
+	EphemeralBackupObject = "cdi.kubevirt.io/ephemeral-backup-object"
+	//InstallStrategyLabel This label indicates the object is a part of the install strategy retrieval process.
+	InstallStrategyLabel = "cdi.kubevirt.io/install-strategy"
```

Comment:
None of these referenced anywhere yet.  I possible see a use for `ManagedByLabel` and `ManagedByLabelOperatorValue` but do we need the rest?

Reply:
removed